src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/models/ReportVersionDetails.kt

```diff
@@ -7,8 +7,6 @@ data class ReportVersionDetails(val name: String,
                                 val id: String,
                                 val published: Boolean,
                                 val date: Instant,
-                                val author: String,
```

Comment:
There didn't seem to be any current need for the custom fields to be on the details class